README.md

```diff
@@ -155,19 +155,17 @@ If this looks strange to you, you can also use the more traditional [blocking AP
 As stated above, this library provides you a powerful, async API by default.
 
 If, however, you want to integrate this into your traditional, blocking environment,
-you should look into also using [clue/reactphp-block](https://github.com/clue/reactphp-block).
+you should look into also using [reactphp/async](https://github.com/reactphp/async).
```

Comment:
In @clue's original PR in [reactphp/http#464](https://github.com/reactphp/http/pull/464/files) he also rewrote the whole Blocking chapter inside the documentation. Can you look into that?

I also did the same thing in [clue/reactphp-mq#34](https://github.com/clue/reactphp-mq/pull/34/files).